coral-component-columnview/src/tests/test.ColumnView.js

```diff
@@ -484,6 +484,20 @@ describe('ColumnView', function() {
         expect(columnActiveChangeSpy.calledBefore(changeSpy)).to.be.true;
       });
 
+      it('should not active when column is selected ', function() {
+        helpers.build(window.__html__['ColumnView.full.html'], function(el) {
```

Comment:
The test doesn't test anything because `helpers.build` doesn't accept a function see the other tests and https://github.com/adobe/coral-spectrum/blob/master/coral-utils/src/tests/helpers/helpers.build.js#L41-L46